src/components/cl/basic/cl_basic_context.c

```diff
@@ -67,9 +67,33 @@ ucc_status_t
 ucc_cl_basic_get_context_attr(const ucc_base_context_t *context, /* NOLINT */
                               ucc_base_ctx_attr_t      *attr)
 {
+    const ucc_cl_basic_context_t *ctx =
+        ucc_derived_of(context, ucc_cl_basic_context_t);
+    ucc_base_ctx_attr_t tl_attr;
+    ucc_status_t        status;
+    int                 i;
+
     if (attr->attr.mask & UCC_CONTEXT_ATTR_FIELD_CTX_ADDR_LEN) {
         attr->attr.ctx_addr_len = 0;
     }
-    // attr->attr.mask & UCC_CONTEXT_ATTR_FIELD_CTX_ADDR - nothing to do
+
+    /* CL BASIC reports topo_rquired if any of the TL available
+       TL contexts needs it */
+    attr->topo_required = 0;
```

Comment:
Why this is a property of TL, it should be exclusively CL, right?

Reply:
discussed in WG: can be property of both  CL/TL depending on the alg implementation and needs